lib/inets/doc/src/http_uri.xml

```diff
@@ -63,8 +63,6 @@
        <item><p>Example: http, https</p></item>
        <tag><c>host() = string() | unicode:unicode_binary()</c></tag>
        <item><p></p></item>
-       <tag><c>port() = pos_integer()</c></tag>
```

Comment:
I think the purpose of the author of this module was to describe the parts of the standard URI (scheme, userinfo, host, port, path, query, fragment). Thus, it would be better to keep 'port' with the type port_number().

Reply:
Addressed in 6251070350e48c0e438cf848e2a44c5c5b73b7ae